stdlib/Random/src/misc.jl

```diff
@@ -60,6 +60,12 @@ julia> randstring(MersenneTwister(3), 'a':'z', 6)
 
 julia> randstring("ACGT")
 "TGCTCCTC"
+
+julia> randstring(12)
+"SHXK1CKamNtK"
+
+julia> randstring(['A':'Z'; '0':'9'], 8)
```

Comment:
This example is redundant with `randstring(MersenneTwister(3), 'a':'z', 6)` above, so I would remove it.

Reply:
Without this example, it is not clear to me how I would have done this (combine two ranges together)

Reply:
I don't think this is the right place to show how to combine two ranges. But I wouldn't mind if you want to replace the example with `'a':'z'` with yours though.

Reply:
That works for me!

Reply:
It may be of interest to use a collection of characters rather than just 'a':'z', for example being able to add characters such as ĩ,ũ,ü,ö,ä,ж,ю may be helpful.

Reply:
@logankilpatrick do you like to update your PR with the suggested change?